### PR TITLE
[Recognizer] Add docker image

### PR DESCRIPTION
**Pull request by mickel8**

### CODE REVIEW

File: recognizer/config/config.exs [hunk 1]
```diff
@@ -55,6 +55,8 @@ config :nx, default_backend: EXLA.Backend
 
 config :recognizer, max_rooms: 5, max_session_time_s: 200
 
+config :bundlex, :disable_precompiled_os_deps, apps: [:ex_libsrtp]
+
```
[the remark this answers is not in the document]
Comment (LVala) [reply recognizer/config/config.exs:59]: And btw, why we don't use the bundled `libsrtp`?

[the remark this answers is not in the document]
Comment (mickel8) [reply recognizer/config/config.exs:59]: There is a bug when doing `mix release`, at least on Arch linux

[the remark this answers is not in the document]
Comment (mickel8) [reply recognizer/config/config.exs:59]: Let's remove this and for testing I will just have a patch locally

**Review by LVala — APPROVED** [review]

File: recognizer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+# https://hub.docker.com/r/hexpm/elixir/tags?page=1&name=ubuntu
+# https://hub.docker.com/_/ubuntu?tab=tags
+#
+# This file is based on these images:
+#
+#   - https://hub.docker.com/r/hexpm/elixir/tags - for the build image
+#   - https://hub.docker.com/_/debian?tab=tags&page=1&name=bullseye-20231009-slim - for the release image
+#   - https://pkgs.org/ - resource for finding needed packages
+#   - Ex: hexpm/elixir:1.16.0-erlang-26.2.1-debian-bullseye-20231009-slim
+#
```
Comment (sgfn) [inline recognizer/Dockerfile:13]: Delete the final `#`

**Review by sgfn — APPROVED** [review]

File: recognizer/lib/recognizer/application.ex [hunk 3]
```diff
@@ -16,9 +16,13 @@ defmodule Recognizer.Application do
...
-      {hash, 0} -> "(#{String.trim(hash)})"
-      _ -> ""
+    try do
+      case System.cmd("git", ["rev-parse", "--short", "HEAD"]) do
+        {hash, 0} -> "(#{String.trim(hash)})"
+        _ -> ""
+      end
+    catch
+      _, _ -> ""
     end
```
[the remark this answers is not in the document]
Comment (mickel8) [reply recognizer/lib/recognizer/application.ex]: Fixed. I also resigned from reading git commit as this is problematic in our docker build (we have Dockerfile nested in recognizer subdirectory so it doesnt have access to the `.git` dir in its parent)

**Pull request merged**